lib/timers.js

```diff
@@ -87,13 +87,20 @@ function listOnTimeout() {
 
   var first;
   while (first = L.peek(list)) {
-    // If the previous iteration caused a timer to be added,
```

Comment:
Agreed, it's probably self-explanatory in the current state of this PR. Thank you for questioning my comment :) :+1: